src/api/response/iiif/manifest.js

```diff
@@ -234,6 +237,24 @@ function transform(response) {
       }
     });
 
+    /**
+     * Add a placeholderCanvas property to a Canvas if the annotation body is of type "Image"
+     * (iiif-builder package currently doesn't support adding this property)
+     */
+    for (let i = 0; i < jsonManifest.items.length; i++) {
+      if (jsonManifest.items[i].items[0].items[0].body.type === "Image") {
```

Comment:
Is it a 100% guarantee that every member of `jsonManifest.items` is going to have a `items[0].items[0].body`, or does this conditional need some `?.` optional chaining?